### PR TITLE
Fix the problem that Tasks&Jamboard&Fitbit App cannot log in with Google

### PR DESCRIPTION
**Pull request by DaVinci9196**
1.Fix the problem that Tasks&Jamboard&Fitbit App cannot log in with Google 
2.Declare CAR_SPEED permission

### CODE REVIEW

File: play-services-core/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -90,6 +90,11 @@
         android:description="@string/perm_provision_description"
         android:label="@string/perm_provision_label"
         android:protectionLevel="privileged|signature" />
+    <permission
+        android:name="com.google.android.gms.permission.CAR_SPEED"
```
Comment (mar-v-in) [inline play-services-core/src/main/AndroidManifest.xml:94]: Which app/issue is fixed by adding the `CAR_SPEED` permission? What about `CAR_MILEAGE` and `CAR_FUEL` permissions?

**Comment by ale5000-git** [comment]
Fitbit isn't a Google apps, also I never had any problem with it.

**Comment by ale5000-git** [comment]
Fitbit isn't a Google apps, also I never had any problem with it.
Tasks correctly use android.permission.GET_ACCOUNTS.
Probably the only needed onw is Jamboard.

File: play-services-core/src/main/AndroidManifest.xml (hunk 1 above)
Comment (DaVinci9196) [reply play-services-core/src/main/AndroidManifest.xml:94]: Fixed the Google Maps location permission authorization pop-up window when connecting to Android Auto. The window still pops up after authorization.

**Comment by ale5000-git** [comment]
@mar-v-in 
If you want I already have a commit with these car related permissions (excluding the signature ones) here: https://github.com/micro5k/GmsCoreMod/commit/6026519041794bf03b68d675860adb3a5aa94cf1.patch

**Comment by ale5000-git** [comment]
How already said Fitbit and Tasks doesn't need this change.

The other parts are superceded by #2214 and #2208.

**Pull request closed without merging**